### PR TITLE
UUID is now cleaned when destroying format

### PR DESCRIPTION
**Pull request by japokorn**
When reformatting UUID is now reset

### CODE REVIEW

**Comment by vpodzime** [comment]
I somehow cannot see how the commit message could be related to the change in the code.

**Comment by japokorn** [comment]
Rephrased commit message to be more clear. (hopefully)

**Comment by japokorn** [comment]
jenkins, test this please

**Comment by vpodzime** [comment]
Looks good to me.

File: blivet/deviceaction.py [hunk 1]
```diff
@@ -690,8 +690,11 @@ def apply(self):
     def execute(self, callbacks=None):
         """ wipe the filesystem signature from the device """
         # remove any flag if set
-        if self.format.parted_flag:
-            self.device.unset_flag(self.format.parted_flag)
+        if isinstance(self.device, PartitionDevice) and self.device.disklabel_supported:
+            if self.format.parted_flag:
+                self.device.unset_flag(self.format.parted_flag)
+            if self.format.parted_system is not None:
+                self.device.parted_partition.system = None
```
Comment (dwlehman) [inline blivet/deviceaction.py]: Don't you need to also do `self.device.disk.commit_to_disk()` or similar, as in `ActionCreateFormat.execute`?

Comment (japokorn) [inline blivet/deviceaction.py]: Fixed.

**Comment by japokorn** [comment]
Pushed.

**Pull request merged**